src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/client/prisonRegisterApi/model/PrisonLite.kt

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.client.prisonRegisterApi.model
```

Comment:
This class seems pointless as you have the Organisation class generated by openAPI which is the same?